lib/nfg/capistrano/config/settings.rb

```diff
@@ -11,7 +11,7 @@
 set :linked_files, fetch(:linked_files, []).push('config/database.yml', 'config/redis.yml', 'config/redis-jobs.yml', 'config/api-keys.yml')
 
 # Default value for linked_dirs is []
-set :linked_dirs, fetch(:linked_dirs, []).push('node_modules', 'log', 'tmp/pids', 'tmp/cache', 'tmp/sockets', 'public/system', 'public/assets')
+set :linked_dirs, fetch(:linked_dirs, []).push('node_modules', 'log', 'tmp/pids', 'tmp/cache', 'tmp/sockets', 'public/system', 'public/assets', 'public/packs')
```

Comment:
https://stackoverflow.com/questions/45806389/what-are-public-packs-and-public-packs-test-for

Reply:
Yes, the contents of `public/packs` is generated by webpacker, which gets run as part of the `assets:precompile` rake task.